textproc/cmconvert/Portfile

```diff
@@ -1,21 +1,36 @@
-PortSystem 1.0
-name	    	cmconvert
-version	    	1.8.6
-categories  	textproc
-maintainers 	stromberg.org:thomas+darwinports
-description    	Converts GPX files with GPS coordinates into into CacheMate PDB files
-long_description    	Converts Geocaching.com GPX files into CacheMate PDB files, so \
-			that they may be used by the CacheMate software for PalmOS.
-homepage	http://www.smittyware.com/palm/cachemate/tools.php
-platforms	darwin
-master_sites	http://www.smittyware.com/download/ \
-		fink:distfiles
-checksums	md5 e269cc710e097cb1a365db550a0940d0
-depends_lib     lib:libexpat.0:expat lib:libzzip.0.10:libzzip
-configure.args  --with-expat-dir=${prefix} --with-zzip-dir=${prefix} \
-		--mandir=${destroot}${prefix}/share/man
-
-destroot.destdir	prefix=${destroot}${prefix}
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                cmconvert
+version             1.9.6
+revision            0
+categories          textproc
+platforms           darwin
+license             GPL-2+
+maintainers         stromberg.org:thomas+darwinports
+
+
+description         Converts GPX files with GPS coordinates into into CacheMate PDB files
+long_description    Converts Geocaching.com GPX files into CacheMate PDB files, so \
+                    that they may be used by the CacheMate software for PalmOS.
+
+homepage            https://www.smittyware.com/palm/cachemate/tools.php
+master_sites        http://www.smittyware.com/download/
+
+checksums           md5     d0d6be5ac88becc80f81cef4229565b4 \
```

Comment:
You can drop `md5`, checksums can simply have just the recommended types, which are `rmd160`, `sha256`, & `size`.